docker-compose.example.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.7"
+
+# NOTE: We don't use Docker in production at Retrospring, so this setup is
+#       provided as-is and might be broken or could be improved.
+#
+#       If you have experience in production docker setups, feel free to
+#       send in some patches!
+
+services:
+  web:
+    image: ghcr.io/retrospring/retrospring:main
```

Comment:
is there a nice way to specify the version to run from the docker-compose CLI?  this would be preferred for production deployments, as `main` changes all the time